qa/L0_perf_resnet/test.sh

```diff
@@ -137,6 +137,13 @@ done
 # Tests with optimization enabled models
 for MODEL_NAME in $OPTIMIZED_MODEL_NAMES; do
     for PROTOCOL in $PROTOCOLS; do
+        # NOTE: Skip TF-TRT for C API test, it is too inconsistent with
```

Comment:
Create a jira ticket to enable TF-TRT optimization in C API test. You can mention several solutions like offline TF-TRT optimization we were discussing.